modules/javafx.graphics/src/main/java/javafx/application/Platform.java

```diff
@@ -432,4 +436,280 @@ public static ReadOnlyBooleanProperty accessibilityActiveProperty() {
         }
         return accessibilityActiveProperty.getReadOnlyProperty();
     }
+
+    /**
+     * Gets the preferences of the current platform.
+     * <p>
+     * The map returned from this method is unmodifiable, which means that keys and values cannot
+     * be added, removed, or updated. Calling any mutator method on the map will always cause
+     * {@code UnsupportedOperationException} to be thrown. However, the mappings will be updated
+     * by JavaFX when the operating system reports that a platform preference has changed.
+     *
+     * @return the {@code Preferences} instance
```

Comment:
minor: make it a link or add a line with a link to the comment itself saying something like
`Please refer to { @ link Preferences } javadoc for a list of expected preferences.`

Reply:
I think there is no need to make this a link, since the return type of the method already provides the link.

Reply:
not in Eclipse.  {@code} shows up in monospace font, while {@link} does the link.

Reply:
Yes, I know that `{@code}` shows up in monospace font, while `{@link}` provides the link. It does that in the javadoc-generated API docs, too.

What I meant by my comment is that since the method in question returns a `Preferences` object, it already generates a link to that class when listing the signature the method. Our usual practice is to not include a link to a class that is returned by a method or passed in as a parameter to a method, since it is a redundant link.

Reply:
All I actually want is a link to the platform keys, not to the Properties class per se.  
Perhaps something similar to what String.format() does.

Reply:
Hmm, maybe. In that case, the description seems the best place for that sort of link, rather than the `@return`. Although, since that list is a predominant part of the class docs of Preferences, the link in the method signature that already goes to Preferences class ought to cover that, I would think. I wouldn't mind either way.

Reply:
It's just a suggestion, really, saves a click.